data-remote/src/main/java/com/teamdontbe/data_remote/api/MyPageUserProfileApiService.kt

```diff
@@ -0,0 +1,17 @@
+package com.teamdontbe.data_remote.api
+
+import com.teamdontbe.data.dto.BaseResponse
+import com.teamdontbe.data.dto.response.ResponseMyPageUserProfileDto
+import retrofit2.http.GET
+import retrofit2.http.Path
+
+interface MyPageUserProfileApiService {
+    companion object {
+        const val VIEW_MEMBER = "viewmember"
+    }
+
+    @GET("/${LoginApiService.API}/${LoginApiService.V1}/$VIEW_MEMBER/{viewmemberId}")
+    suspend fun getMyPageUserProfile(
+        @Path("viewmemberId") viewMemberID: Int,
```

Comment:
```suggestion
        @Path("viewmemberId") viewMemberID: Int,
```
p1 : viewmemberId 요 아이도 상수화 할 수 있슴다